Makefile

```diff
@@ -23,6 +23,12 @@ CC           = /usr/local/opt/llvm/bin/clang
 CXX          = /usr/local/opt/llvm/bin/clang++
 CGC          = $(NXDK_DIR)/tools/cg/mac/cgc
 endif
+ifneq (,$(findstring MSYS_NT,$(UNAME_S)))
+LD           = lld-link
+CC           = clang
+CXX          = clang++
+CGC          = $(NXDK_DIR)/tools/cg/win/cgc
```

Comment:
I think we should include both and select on the running environment (this would be a seperate change / PR).
Ideally we'd have our own `cgc` like tool in open-source (but we don't have developer capacity for this).

However, 32 bit is almost dead and we don't know how long Microsoft will support 32 bit still.
So 64 bit only is fine. People can create an issue if they run into problems.
(No need to waste time on changing this now)